lib/utils/extension/build_context_extension.dart

```diff
@@ -137,4 +138,9 @@ extension ContextExtensionss on BuildContext {
       return mobile!;
     }
   }
+
+  void goChild(String path) => go('${GoRouterState.of(this).uri.path}$path');
+
+  void pushChild(String path) =>
+      push('${GoRouterState.of(this).uri.path}$path');
```

Comment:
Why dont you `push('${GoRouterState.of(this).uri.path}/$path');`

Reply:
no need,

Reply:
i mean, so that you can use `pushChild('invite')` instead of `pushChild('/invite')`

Reply:
good idea